### DISCUSSION TITLE
Attendee survey

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#16] [Ideas]
Following from #15, we have said that we want to survey the attendees before the first day. This thread can be used to suggest the questions we want to ask:

1. Pick 3 breakout room topics you are interested in (and also suggest more on GitHub for the "wildcard" sessions nearer the end of the workshop)
2. What stages of the research data cycle do you primarily work on? (This could be as a researcher directly, or as a tool builder)
3. Which of the following technologies do you make use of in your work on research data management? (open file formats, API specifications and schemas, ontologies and linked data, graph databases, document databases, relational databases, git, GitHub)
4. Are you interested in contributing to communal open source software libraries that could be used in the chemistry/materials research data management community?
5. Do you know of a library that you think should be more widely used and supported in the community?

**Comment by kjappelbaum** [comment]
I'd consider asking about user stories. They could be a great starting point for discussions and if we have some before the workshop we can tailor the technical discussions around those, i.e., they might tell us something about the technologies we need.

For instance, "how could increased interoperability help your research"?

**Comment by kjappelbaum** [comment]
I'd also consider asking about what they currently consider the biggest limitations/problems. Is it the tooling? (Perceived) lack of standards? Lack of incentives?

**Comment by sphuber** [comment]
> Do you know of a library that you think should be more widely used and supported in the community?

Maybe in the light of another discussion of existing standards not getting traction, we could add if they know of formats/schemas/ontologies/frameworks that could be of interest.

**Comment by ml-evs** [comment]
Any ideas on how to do the survey? I guess something live like Mentimeter won't work if we are planning on screening the talk to multiple timezones... anyone want to take charge of this?

**Reply by kjappelbaum** [reply]
TBH, I would simply make a Google Forms for the pre-workshop survey (and send it one week in advance such that we can still adjust things). 

If no one else volunteers I can take care of this ...

**Reply by sphuber** [reply]
I can take care of that one

**Comment by ml-evs** [comment]
Not sure how to phrase it, but a question about whether people use:

1) Proprietary/commerical data management solutions
2) Open source/community solutions
3) Bespoke software written for your research group (e.g., custom web apps, schemas)
4) Domain-specific software (e.g., cheminfo, chemmotion ELNs)
5) More field-agnostic data management software (e.g., eLabFTW)


as parts of their workflows

**Comment by shyamd** [comment]
As a corollary to Number 2. It might be useful to know what stags of the data cycle the participant is interested in beyond what they primarily work on.

**Comment by ml-evs** [comment]
Logistics questions to ask:

1) Would you be interested in co-writing a perspective paper based on this workshop?
2) Would you be interested in attending future remote single-day discussions/hackathon-style events?

These could also be added to the post-workshop survey.

**Reply by sphuber** [reply]
For 1. would there be a potential risk that we get too many yesses? At some point, having too many people could maybe become counterproductive?

**Reply by kjappelbaum** [reply]
from past experience people will not stay around for doing anything (not even say random things in the discussion). However, they might still want to be co-authors. We can be quite clear upfront and state that being co-author requires substantial contribution to the discussion.

**Reply by sphuber** [reply]
I think these two questions would make most sense indeed for a post-workshop survey. Don't think it would be problematic to send a second one after the workshop.

**Reply by ml-evs** [reply]
Yeah, we definitely need a post-workshop survey (I think Sara wanted feedback from attendees too). I guess it depends what timescale we want to operate on with the paper. Preprint up by the end of the workshop week? 😅

**Comment by ml-evs** [comment]
We should also test the interest in the BYOC/BYOD/tutorial sessions for less expert attendees, something like:

Which of the following interactive sessions would interest you?

- [ ] BYOC: present a lightning talk on your software library or tool and ask experts how to improve interoperability, or advertise how your tool can be used to improve interoperability
- [ ] BYOD: for a dataset you have published or are about to publish, receive feedback on how FAIR/interoperable it is, and how it could be improved
- [ ] Python packaging tutorial by Charles
- [ ] Introduction to version control with git (I have a 15 minute canned tutorial and some material for this)
- [ ] Any other informal/"taster" tutorials we can give ourselves or convince others to
